root/edit/details/AddArtist.js

```diff
@@ -126,7 +126,7 @@ const AddArtist = ({edit}: Props): React$MixedElement => {
         ) : null}
 
         <tr>
-          <th>{addColonText(lp('Ended', 'artist ended boolean'))}</th>
+          <th>{addColonText(lp('Ended', 'artist'))}</th>
```

Comment:
The reason I specified boolean here is because there's also "Ended" as a label for end date for artists in some cases, so it feels like it might be confusing? But if you think it's fine, I can live with this.

Reply:
Just understood your point by reading my own comment and the linked commit again. Reverted my edit.